editor/editor_inspector.cpp

```diff
@@ -2767,7 +2767,9 @@ void EditorInspector::update_tree() {
 					// Update the docs reference and the label based on the script.
 					Vector<DocData::ClassDoc> docs = scr->get_documentation();
 					if (!docs.is_empty()) {
-						doc_name = docs[0].name;
+						// The documentation of a GDScript's main class is at the end of the array
+						// Hacky because this isn't necessarily always guaranteed
```

Comment:
```suggestion
						// The documentation of a GDScript's main class is at the end of the array.
						// Hacky because this isn't necessarily always guaranteed.
```
Same below.

Reply:
Done! :)